Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -26,7 +26,7 @@ Write-Verbose "initializationScriptPath = $initializationScriptPath"
 Write-Verbose "runPowershellInParallel = $runPowershellInParallel"
 Write-Verbose "sessionVariables = $sessionVariables"
 
-. ./PowerShellJob.ps1
+. $PSScriptRoot/PowerShellJob.ps1
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
```

Comment:
Refer the following tests to understand how to write function based tests

https://github.com/Microsoft/vsts-rm-extensions/blob/master/Extensions/IISWebAppDeploy/Tests/Tasks/IISWebAppMgmt/ManageIISWebApp.Tests.ps1